frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-page.tsx

```diff
@@ -94,14 +111,21 @@ const InstallCluster: React.FC<InstallClusterProps> = ({ match }) => {
                 id="radio-2"
                 label="Independent - For external storage"
                 name="independent-mode"
+                className="ceph-install--no-margin"
               />
             </div>
           </div>
         )}
         {(isIndependent === false || mode === MODES.CONVERGED) && (
           <CreateOCSServiceForm match={match} />
         )}
-        {mode === MODES.INDEPENDENT && <InstallExternalCluster match={match} />}
+        {mode === MODES.INDEPENDENT && independentReqdKeys && downloadFile && (
```

Comment:
`downloadFile` should not be a hard requirement.

Reply:
Not doing this is a source of bug.